_rules/__tests__/hyperlinks.js

```diff
@@ -0,0 +1,19 @@
+const describeRule = require('../../test-utils/describe-rule')
+const markdownLinkExtractor = require('markdown-link-extractor')
+const isUrl = require('is-url')
+
+const blacklistOfLinks = ['w3.org/TR/dom41', 'WCAG20']
```

Comment:
TODO: 

Add more links to blacklist
- https://www.w3.org/TR/wai-aria-1.0/, these should point to 1.1

Reply:
Ensure this test covers all markdown pages

Reply:
I'm not sure of blacklisting "wcag20" outright. I'd blacklist the following instead:

://www.w3.org/TR/WCAG20/
://www.w3.org/TR/UNDERSTANDING-WCAG20/
://www.w3.org/TR/WCAG20-TECHS/
://www.w3.org/TR/wai-aria-1.0/